src/plugins/orangehrmPerformancePlugin/config/routes.yaml

```
@@ -12,6 +12,55 @@ apiv2_performance_kpis:
   defaults:
     _api: OrangeHRM\Performance\Api\KpiAPI
 
+apiv2_performance_performance_trackers:
+  path: /api/v2/performance/performance-trackers
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET]
+  defaults:
+    _api: OrangeHRM\Performance\Api\PerformanceTrackerAPI
+
+apiv2_performance_performance_trackerId:
+  path: /api/v2/performance/performance-tracker/{id}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, PUT]
+  defaults:
+    _api: OrangeHRM\Performance\Api\PerformanceTrackerAPI
+
+apiv2_performance_save_performance_tracker:
+  path: /api/v2/performance/performance-tracker
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ POST]
+  defaults:
+    _api: OrangeHRM\Performance\Api\PerformanceTrackerAPI
+
+apiv2_performance_delete_performance_trackers:
+  path: /api/v2/performance/performance-trackers
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ DELETE]
+  defaults:
+    _api: OrangeHRM\Performance\Api\PerformanceTrackerAPI
+
+apiv2_performance_reviewer_autocomplete:
+  path: /api/v2/performance/reviewers
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET ]
+  defaults:
+    _api: OrangeHRM\Performance\Api\PerformanceTrackerReviewerAPI
+
+apiv2_performance_my_trackers:
+  path: /api/v2/performance/myTracker
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET ]
+  defaults:
+    _api: OrangeHRM\Performance\Api\MyTrackerAPI
+
+apiv2_performance_my_reviews:
+  path: /api/v2/performance/myReview
```

```suggestion
  path: /api/v2/performance/reviews
```
try to follow [this](https://gist.github.com/RajithaKumara/6f148638e8a03a861c0af9f020a1cd56#file-routes-md) pattern 

use camelCase URL for screens, kebab-case for APIs